### ISSUE TITLE
Error: Cannot set property 0 of undefined

### ISSUE THREAD

**Issue by MrCrimp** [radekmie/react-timepicker#2]
Steps to reproduce:
Chrome

1) Set militaryTime = False
2) Open the time picker (should open in 12 hr mode)
3) Click on 12.

-->Throws an error: "Cannot set property 0 of undefined " in the console

In the provided  example html, this will recreate the bug
```
 ReactDOM.render(React.createElement(Timepicker, {
                    militaryTime: false,
                    hours:   new Date().getHours(),
                    minutes: new Date().getMinutes()
                }), root);
```

**Comment by radekmie** [comment]
OK, I'll take a look at this later.

**Comment by radekmie** [comment]
Fixed in `v1.1.6`.
